ngraph/frontends/neon/optimizer.py

```diff
@@ -363,3 +363,48 @@ def variable_update(self, variable, grad, scale_factor):
                       variable - (scale_factor * self.ell * m) / (ng.sqrt(v) + self.epsilon))
         ])
         return updates
+
+
+class Adagrad(LearningRateOptimizer):
+    """
```

Comment:
Could you add a docstring? You can use this one as a good reference: https://github.com/NervanaSystems/neon/blob/master/neon/optimizers/optimizer.py#L637

Reply:
Sure, I'll do that

Reply:
Done. Updated. Thanks.